### DISCUSSION TITLE
Boostrap Npm deprecated

### DISCUSSION THREAD

**Discussion by Apoorslime** [twbs/bootstrap#37334] [Q&A]
`# Deprecated Package

This package is no longer supported and has been deprecated. To avoid malicious use, npm is hanging on to the package name.

Please contact support@npmjs.com if you have questions about this package. `

Will it be available again?, Will you solve the problem?

**Comment by mdo** [comment]
Perhaps you've spelled it wrong? https://www.npmjs.com/package/bootstrap

**Reply by Apoorslime** [reply]
No

**Reply by Apoorslime** [reply]
Idk it's correct

**Comment by Apoorslime** [comment]
This is package.json

`
{
  "name": "boostrap",
  "version": "2.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/npm/deprecate-holder.git"
  },
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/npm/deprecate-holder/issues"
  },
  "homepage": "https://github.com/npm/deprecate-holder#readme",
  "dependencies": {}
}
`

**Comment by Apoorslime** [comment]
This is README.md

`# Deprecated Package

This package is no longer supported and has been deprecated. To avoid malicious use, npm is hanging on to the package name.

Please contact support@npmjs.com if you have questions about this package. `

**Comment by Apoorslime** [comment]
~/.../projects/my-app $ npm i bootstrap

up to date, audited 1452 packages in 17s          
214 packages are looking for funding
  run `npm fund` for details                      
6 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
                                                  Run `npm audit` for details.
~/.../projects/my-app $ npm install bootstrap@v5.2.2
npm WARN deprecated boostrap@2.0.0: Package no longer supported. Contact support@npmjs.com for more info.                                             
added 1 package, and audited 1452 packages in 19s 
214 packages are looking for funding
  run `npm fund` for details

6 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

**Comment by nkdas91** [comment]
Hi @apoorslime 

Uninstalling ```boostrap``` should solve your problem.

```console
npm uninstall boostrap
```
```console
npm install bootstrap
```

**Reply by Apoorslime** [reply]
it always tells me that it is Deprecated

**Reply by Apoorslime** [reply]
Usage boostrap-react package

**Reply by WalberMelo** [reply]
1. npm uninstall bootstrap
2. npm install react-bootstrap bootstrap
